rnd/autogpt_builder/src/components/admin/marketplace/AdminMarketplaceAgentList.tsx

```diff
@@ -0,0 +1,26 @@
+import { Agent } from "@/lib/marketplace-api";
+import AdminMarketplaceCard from "./AdminMarketplaceCard";
+import { ClipboardX } from "lucide-react"; // Import the ClipboardX icon from lucide-react
```

Comment:
nit: pointless comment looks like by AI

Reply:
Yeah had cursor extract it out and I guess it liked that comment lol

Reply:
fixed